public/app/features/dashboard-scene/sharing/ExportButton/ExportAsImage.tsx

```diff
@@ -0,0 +1,294 @@
+import { css } from '@emotion/css';
+import { saveAs } from 'file-saver';
+import { useState, useEffect } from 'react';
+import { useMeasure } from 'react-use';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { config } from '@grafana/runtime';
+import { SceneComponentProps } from '@grafana/scenes';
+import { Button, Field, LoadingBar, RadioButtonGroup, Alert, useStyles2 } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+import { shareDashboardType } from 'app/features/dashboard/components/ShareModal/utils';
+import { DashboardInteractions } from 'app/features/dashboard-scene/utils/interactions';
+import { getDashboardSceneFor } from 'app/features/dashboard-scene/utils/utils';
+
+import { ShareExportTab } from '../ShareExportTab';
+
+import { generateDashboardImage, ImageGenerationError } from './utils';
+
+enum ImageFormat {
+  PNG = 'png',
+  JPG = 'jpg',
```

Comment:
Do we really want to support JPG? Currently, this option is not working (it generates a PNG whatever option is selected) but I don't think it's worth supporting.